doc/source/reference/c-api.types-and-structures.rst

```diff
@@ -870,6 +894,21 @@ PyUFunc_Type
 
        Override the default nditer flags for the ufunc.
 
+   Added in API version 0x0000000D
+
+   .. c:member:: npy_intp *PyUFuncObject.core_dim_sizes
+
+       For each distinct core dimension, the possible
+       :ref:`frozen <frozen>` size (``-1`` if not frozen)
```

Comment:
Why are we using both `-1` and `UFUNC_CORE_DIM_SIZE_INFERRED` to indicate this?

Reply:
@mhvk ?

Reply:
I think that's a leftover from the initial work by Jaime. I think it is fine to remove the guarantee that it is `-1` if not frozen, as indeed the flag already indicates that (and perhaps we want to use negative numbers to indicate something else in the future).